### ISSUE TITLE
Local Server Manager

### ISSUE THREAD

**Issue by yeslayla** [loudsmilestudios/TetraControl#8]
**Description**

Currently the only implementation of the server manager is for running on AWS. It would be good for testing and general usage purposes if there was an non-ECS version that maybe ran locally by passing in different flags to a server executable for configuring the port.

This would make it easier for testing the core part of the program.

**Comment by antonov314** [comment]
> **Description**
> 
> Currently the only implementation of the server manager is for running on AWS. It would be good for testing and general usage purposes if there was an non-ECS version that maybe ran locally by passing in different flags to a server executable for configuring the port.
> 
> This would make it easier for testing the core part of the program.

Yes Please make that, i really like to run my own server just local.
